index.html

```diff
@@ -3,48 +3,42 @@
     <head>
         <meta charset="UTF-8">
         <link rel="stylesheet" href="css/styles.css">
-        <link rel="stylesheet" href="css/all.min.css">
         <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
         <title>Apple by Bruna</title>
     </head>
     <body>
         <nav>
-            <div class="nav-bar-outer">
-                <div class="nav-bar-inner">
-                    <ul class="nav-bar-links">
-                        <il><a href="#"><i class="fab fa-apple"></i></a></il>
-                        <il><a href="#">Store</a></il>
-                        <il><a href="#">Mac</a></il>
-                        <il><a href="#">iPod</a></il>
-                        <il><a href="#">iPhone</a></il>
-                        <il><a href="#">iPad</a></il>
-                        <il><a href="#">iTunes</a></il>
-                        <il><a href="#">Support</a></il>
-                    </ul>
-                    <div class="search-bar">
-                        <input type="text" id="search-bar">
-                    </div>
+            <div class="container-links">
+                <ul class="nav-bar-links">
+                    <li class="logo"><a href="#"><img src="https://png2.kisspng.com/sh/bcf01b04623291eb8fede18b128ff7c9/L0KzQYm3VMI2N5VvfZH0aYP2gLBuTfFxeJ1qRd54Z3AwfbLqgv9wc151itG2aYDrf7BsTcVibWE5fdU7M0K7RrW6TsEzPWM4UasBMUW1RIe7WcY3PmM1T5D5bne=/kisspng-apple-logo-macbook-pro-iphone-5ae04ec23286d3.125239961524649666207.png" alt="Apple Logo"></a></li>
+                    <li><a href="#">Store</a></li>
+                    <li><a href="#">Mac</a></li>
+                    <li><a href="#">iPod</a></li>
+                    <li><a href="#">iPhone</a></li>
+                    <li><a href="#">iPad</a></li>
+                    <li><a href="#">iTunes</a></li>
+                    <li><a href="#">Support</a></li>
+                </ul>
+                <div class="search-bar">
+                    <input type="text" id="search-bar">
                 </div>
             </div>
         </nav>
-        <article class="billbord">
```

Comment:
You could enclose the dominant content of your body within `<main>` tags :smiley: